tgui/packages/tgui/interfaces/NtosSpaceBetting.tsx

```diff
@@ -0,0 +1,263 @@
+                                    money_betting: value,
+                                  })
+                                }
+                              />
+                            ) : (
+                              <Button.Checkbox
+                                tooltip="Whether this answer won."
+                                checked={Winner === option_name}
+                                key={option_name}
+                                onClick={() => set_winner(option_name)}
+                              />
+                            )}
+                          </Stack.Item>
+                        </Stack.Item>
+                      </Stack.Item>
+                    ),
+                  )}
+                  {!!owner &&
+                    (!locked ? (
+                      <Stack.Item>
+                        <Button.Confirm
+                          fluid
+                          icon="minus"
+                          tooltip="Lock the ability to place/retract bets. This is irreversible!"
+                          onClick={() =>
+                            act('lock_betting', { bet_selected: name })
+                          }
+                        >
+                          Lock Betting
+                        </Button.Confirm>
+                      </Stack.Item>
+                    ) : (
+                      <Button.Confirm
+                        fluid
+                        icon="plus"
+                        tooltip="Finalize results as the checked answer being the winner."
+                        onClick={() =>
+                          act('select_winner', {
+                            bet_selected: name,
+                            winning_answer: Winner,
+                          })
+                        }
+                      >
+                        Finalize Results
+                      </Button.Confirm>
+                    ))}
+                </Stack.Item>
+                <Stack.Item>
+                  <Button
+                    fluid
+                    icon="minus"
+                    disabled={locked}
+                    tooltip="If you have any bets, this will remove them and refund the money."
+                    onClick={() => act('cancel_bet', { bet_selected: name })}
+                  >
+                    Cancel Bet
+                  </Button>
+                </Stack.Item>
+              </Stack>
+            </Section>
+          ),
+        )
+      )}
+    </Section>
+  );
+};
+
+export const BettingCreation = () => {
+  const { act, data } = useBackend<Data>();
+  const { max_title_length, max_description_length } = data;
+  const [Title, set_title] = useState('');
+  const [Desc, set_desc] = useState('');
+  const [Option1, set_option1] = useState('');
+  const [Option2, set_option2] = useState('');
+  const [Option3, set_option3] = useState('');
+  const [Option4, set_option4] = useState('');
```

Comment:
these should be camelCased

Reply:
I misread the previous one, if you want it how it was before then what was the point of the og comment

Reply:
it's `[option1, setOption1]`

Reply:
THATS WHAT IT WAS